<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskMainToolEditDlg</class>
 <widget class="QDialog" name="TaskMainToolEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1024</width>
     <height>680</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QWidget" name="widgetTool" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>680</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QWidget" name="widgetPre" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>680</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widgetRL" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>680</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>关联定位</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widgetComm" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>680</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>746</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>基准图像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectImage">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>726</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>未处理图像1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>未处理图像2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>预处理后图像</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>726</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPicName">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>726</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>726</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOSDSet">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>726</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OSD设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>726</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>726</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcute">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>726</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTaskName">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>8</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.直线定位</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>721</y>
     <width>1024</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>1024</width>
     <height>35</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>直线定位参数</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>预处理</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>关联定位</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>公共</string>
    </attribute>
   </widget>
  </widget>
  <widget class="StepEditExcuteLabel" name="labelExecPic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBoxSelectImage</zorder>
  <zorder>btnPrev</zorder>
  <zorder>txtPicName</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnOSDSet</zorder>
  <zorder>btnNext</zorder>
  <zorder>btnQuit</zorder>
  <zorder>btnExcute</zorder>
  <zorder>line_2</zorder>
  <zorder>labelTaskName</zorder>
  <zorder>labelExecPic</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StepEditExcuteLabel</class>
   <extends>QLabel</extends>
   <header>Tool/MainPanel/Roi/StepEditExcuteLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
